Source:
Internal
Info:
This patch is created so that objcopy will support additional targets.
Status:
Submitted upstream. Emailed binutils@sourceware.org and submitted
bug at https://sourceware.org/bugzilla/show_bug.cgi?id=19501.
2.26 added more targets but we still need _pe_.

diff -wpruN '--exclude=*.orig' a~/bfd/config.bfd a/bfd/config.bfd
--- a~/bfd/config.bfd	1970-01-01 00:00:00
+++ a/bfd/config.bfd	1970-01-01 00:00:00
@@ -567,13 +567,13 @@ case "${targ}" in
   i[3-7]86-*-solaris2*)
     targ_defvec=i386_elf32_sol2_vec
     targ_selvecs="iamcu_elf32_vec i386_coff_vec i386_pei_vec"
-    targ64_selvecs="x86_64_elf64_sol2_vec l1om_elf64_vec k1om_elf64_vec x86_64_pei_vec"
+    targ64_selvecs="x86_64_elf64_sol2_vec l1om_elf64_vec k1om_elf64_vec x86_64_pei_vec i386_pe_vec x86_64_pe_vec"
     want64=true
     ;;
 #ifdef BFD64
   x86_64-*-solaris2*)
     targ_defvec=i386_elf32_sol2_vec
-    targ_selvecs="x86_64_elf64_sol2_vec l1om_elf64_vec k1om_elf64_vec iamcu_elf32_vec i386_coff_vec i386_pei_vec x86_64_pei_vec"
+    targ_selvecs="x86_64_elf64_sol2_vec l1om_elf64_vec k1om_elf64_vec iamcu_elf32_vec i386_coff_vec i386_pei_vec x86_64_pei_vec i386_pe_vec x86_64_pe_vec"
     want64=true
     ;;
 #endif
